package.json

```diff
@@ -81,7 +95,8 @@
     "react-syntax-highlighter": "^15.5.0",
     "standard-version": "^9.3.2",
     "tailwindcss": "^3.0.23",
-    "typescript": "^4.6.3"
+    "typescript": "^4.6.3",
+    "webpack": "5"
```

Comment:
why do we need `webpack` as a dev dependency?

Reply:
It is overwriting some indirect dependency that includes the webpack 4. Storyboard just advised me do do it in order to avoid some issues.